### ISSUE TITLE
ressources

### ISSUE THREAD

**Issue by Knight1** [hetznercloud/terraform-provider-hcloud#16]
I can understand that changing the user_data forces the resource to be recreated.
But the _different_ datacenter name should not do that.

```      
      id:           "936705" => <computed> (forces new resource)
      datacenter:   "nbg1-dc3" => "2" (forces new resource)
```

**Comment by LKaemmerling** [comment]
Currently you can not move a instance from one datacenter to another. So the process migration one instance to another datacenter is creating a new resource and make the migration "manually".

**Comment by Knight1** [comment]
This is not what i meant. I put the word different not by accident cursive.

```hcloud datacenter list
ID   NAME        DESCRIPTION          LOCATION
1    fsn1-dc8    Falkenstein 1 DC 8   fsn1
2    nbg1-dc3    Nuremberg 1 DC 3     nbg1
3    hel1-dc2    Helsinki 1 DC 2      hel1
4    fsn1-dc14   Falkenstein 1 DC14   fsn1
```

nbg1-dc3 is datacenter ID 2 ;)
So it should not create a new ressource :D

**Comment by LKaemmerling** [comment]
@Knight1 
TL;DR
We decided to not implement this, because it is not worth the work. 

Long:
The problem on this part is, that we suggest, to use the datacenter name, not the ID. In the most cases, the name is manny more readable than the id. So we think, this could be confusing for some peoples.
